docs/source/actions.rst

```diff
@@ -165,6 +165,17 @@ SPIDERMON_EMAIL_SUBJECT_TEMPLATE
 Slack action
 ============
 
+.. warning::
+
+   You need to install `slackclient`_ to use this feature.
+
+   .. code-block:: shell
+
+     $ pip install "slackclient>=1.3,<2.0"
+
+   **Do not install** a version greater than **1.3.*** because newer versions
```

Comment:
There could be maintenance releases before 2 which will still work.
Apart from this everything looks good.